### DISCUSSION TITLE
Running

### DISCUSSION THREAD

**Discussion by jatinkalwar** [GitSquared/edex-ui#1132] [Questions]
its installed in my win 7 32 bit very easily without any problem but this is not opening in my pc when i click on open....(HELP)

**Comment by GitSquared** [comment]
Sorry, we don't support Windows 7.

**Reply by jatinkalwar** [reply]
How can I use this win 7 any solution??

**Reply by GitSquared** [reply]
Windows 7 has been EOL'ed (End of Life) since January 2020. You need to upgrade to a more recent operating system (also for your own safety, as outdated software is prone to malware & security issues).

**Reply by jatinkalwar** [reply]
Can I install win 10 in 32bit processor system?

**Reply by GitSquared** [reply]
Yes, it should work!

**Comment by jatinkalwar** [comment]
![image](https://user-images.githubusercontent.com/77274095/123533343-491e6f00-d732-11eb-969f-5291bb4936f8.png)
Why it's showing offline in world map and world traffic?? Solve this pls

**Reply by GitSquared** [reply]
Looks like your computer is having trouble loading all the UI correctly. Your hardware might just not be powerful enough...
